app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/CoinItem.kt

```diff
@@ -9,26 +9,34 @@ import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.constraintlayout.compose.ConstraintLayout
 import androidx.constraintlayout.compose.Dimension
 import co.nimblehq.compose.crypto.R
-import co.nimblehq.compose.crypto.extension.toFormattedString
-import co.nimblehq.compose.crypto.ui.common.price.PriceChange
+import co.nimblehq.compose.crypto.core.common.price.PriceChange
+import co.nimblehq.compose.crypto.core.extension.toFormattedString
+import co.nimblehq.compose.crypto.core.theme.*
+import co.nimblehq.compose.crypto.core.uimodel.CoinItemUiModel
+import co.nimblehq.compose.crypto.domain.model.CoinItem
 import co.nimblehq.compose.crypto.ui.preview.CoinItemPreviewParameterProvider
-import co.nimblehq.compose.crypto.ui.theme.*
-import co.nimblehq.compose.crypto.ui.uimodel.CoinItemUiModel
 import coil.compose.rememberAsyncImagePainter
 
+const val TestTagCoinItemSymbol = "CoinItemCoinSymbol"
+const val TestTagCoinItemCoinName = "CoinItemCoinName"
+const val TestTagCoinItemPrice = "CoinItemPrice"
+const val TestTagCoinItemPriceChange = "CoinItemPriceChange"
+
 @Composable
 fun CoinItem(
+    modifier: Modifier = Modifier,
     coinItem: CoinItemUiModel,
     onItemClick: () -> Unit
```

Comment:
`Modifier` should come after other parameters (which doesn't have default value)
```suggestion
    coinItem: CoinItemUiModel,
    onItemClick: () -> Unit,
    modifier: Modifier = Modifier
```